style/src/main/kotlin/ktx/style/style.kt

```diff
@@ -84,8 +93,45 @@ inline operator fun <reified Resource : Any> Skin.set(name: String, resource: Re
  * @param name name of the passed resource.
  * @param resource will be added to the skin and mapped to the selected name.
  */
-inline operator fun <reified Resource : Any , E : Enum<E>> Skin.set(name: E, resource: Resource) =
-        this.set(name.toString(), resource)
+inline operator fun <reified Resource : Any, E : Enum<E>> Skin.set(name: E, resource: Resource) =
+    this.set(name.toString(), resource)
+
+/**
+ * Utility function that makes it easier to add [Skin] assets.
+ * @param name name of the passed resource. Defaults to [defaultStyle].
+ * @param resource will be added to the skin and mapped to the selected name.
+ */
+inline fun <reified Resource : Any> Skin.add(resource: Resource, name: String = defaultStyle) =
```

Comment:
Don't change the order of the parameters in relation to the original skin method.

Reply:
I changed the order so the method can be called with the default name:
```kotlin
skin.add(resource)
```
Instead of
```kotlin
skin.add(resource = resource)
```
Sure someone could just use the `+=` method instead but personally I sometimes try to avoid overloaded operators.

Reply:
OK, makes sense.